Client/Frontend/Browser/Playlist/PlaylistSharedFolder.swift

```diff
@@ -0,0 +1,195 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import Data
+import CoreData
+
+public struct PlaylistSharedFolderModel: Codable {
+  public let version: String
+  public let playlistId: String
+  public let folderName: String
+  public let folderImage: URL
+  public let creatorName: String
+  public let creatorLink: String
+  public let updateAt: String
+  public var mediaItems: [PlaylistInfo]
+  
+  public init(playlistId: String) {
+    version = "1"
+    self.playlistId = playlistId
+    folderName = ""
+    folderImage = NSURL() as URL
```

Comment:
why do we need the NSURL bridge here?

Reply:
Fixed (removed): https://github.com/brave/brave-ios/pull/5798/commits/db40c42a11edc78866fbec8178ed48483df5761e